src/test/scala/com/atomist/rug/runtime/js/EditorTest.scala

```diff
@@ -442,33 +497,64 @@ class EditorTest extends FlatSpec with Matchers {
     assert(p.getMaxLength === 100)
   }
 
-  private  def invokeAndVerifyConstructed(tsf: FileArtifact): (JavaScriptProjectEditor, SuccessfulModification) = {
+  it should "load empty repo in generator" in {
+    val tsf = SimpleFileBasedArtifactSource(
+      StringFileArtifact(".atomist/generators/Thing.ts", SimpleGeneratorLoadingEmptyRepo),
+      StringFileArtifact("I dont", "want this"))
+    val as = TypeScriptBuilder.compileWithModel(tsf)
+
+    val jsed = RugArchiveReader(as).generators.head.asInstanceOf[JavaScriptProjectGenerator]
+
+    val prj = jsed.generate("woot", SimpleParameterValues(Map("content" -> "Anders Hjelsberg is God")))
+    jsed
```

Comment:
What is the point of this last statement?

Should you add a test to make sure the "I dont" file does not exist in the generated project?